tests/instruments/thorlabs/test_elliptec_base.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""
+Created the 20/06/2023
+
+@author: Sebastien Weber
+"""
+import pytest
+
+from pymeasure.test import expected_protocol
+from pymeasure.instruments.thorlabs.elliptec_utils.base import ElliptecController, MotorChannel
+
```

Comment:
You asked for a protocol test with a channel:

```suggestion
def test_motor_position():
    with expected_protocol(
            ElliptecController,
            [("5gp", "7.9"), ("5ma45", None)],
    ) as inst:
        assert inst.ch_5.position == 7.9
        inst.ch_5.position = 45
```

The protocol tests work normally: You acess the channel in the same manner as usual and write the communication as it happens. You might use my TestGenerator (#882), if you whish so.